com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/shader/ShaderCompilePipelineLegacy.java

```diff
@@ -236,6 +236,8 @@ public byte[] crossCompile(ShaderDesc.ShaderType shaderType, ShaderDesc.Language
         } else if(shaderLanguage == ShaderDesc.Language.LANGUAGE_WGSL) {
             String result = compileSPIRVToWGSL(module.spirvResult.source, this.pipelineName);
             return result.getBytes();
+        } else if(shaderLanguage == ShaderDesc.Language.LANGUAGE_HLSL) {
+            return this.generateCrossCompiledShader(shaderType, shaderLanguage, 50);
```

Comment:
TODO: we should remove all specific enums for the graphics languages (e.g LANGUAGE_GLSL_SM120) and just use LANGUAGE_GLSL/HLSL/SPIRV plus a version